src/dgnss_management.c

```diff
@@ -291,8 +292,17 @@ void dgnss_update(u8 num_sats, sdiff_t *sdiffs, double receiver_ecef[3])
   double ref_ecef[3];
   if (num_sats >= 5) {
     double b2[3];
-    least_squares_solve_b_external_ambs(nkf.state_dim, nkf.state_mean,
-        sdiffs_with_ref_first, dd_measurements, receiver_ecef, b2);
+    s8 code = least_squares_solve_b_external_ambs(nkf.state_dim, nkf.state_mean,
+        sdiffs_with_ref_first, dd_measurements, receiver_ecef, b2,
+        disable_raim, raim_threshold);
+
+    if (code < 0) {
+      log_warn("dgnss_update. baseline estimate error: %d\n", code);
+      /* Use b = 0, continue */
```

Comment:
to clear arrays we mostly use `memset(b2, 0, sizeof(b2));`, this is fine but perhaps change for consistency.